WORKSPACE

```diff
@@ -60,7 +60,7 @@ maven_install(
     repositories = ["https://repo.maven.apache.org/maven2/"],
 )
 
-_gapic_generator_java_version = "2.28.1-SNAPSHOT"  # {x-version-update:gapic-generator-java:current}
+_gapic_generator_java_version = "2.28.1"  # {x-version-update:gapic-generator-java:current}
```

Comment:
Somehow release-please doesn't change this version, I have to change it manually.